src/core/trtserver.h

```diff
@@ -336,6 +336,14 @@ TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_ServerModelStatus(
     TRTSERVER_Server* server, TRTSERVER_Protobuf** status,
     const char* model_name);
 
+// Load the requested model.
+TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_LoadModel(
+    TRTSERVER_Server* server, const char* model_name);
+
+// Unload the requested model.
+TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_UnloadModel(
```

Comment:
Does not return until model is unloaded? Returned error indicates if model unloaded successfully or not?